packages/types/api.d.ts

```diff
@@ -1488,7 +1488,7 @@ export interface ITokenConfig {
    * For example see: https://github.com/chevrotain/chevrotain/tree/master/examples/lexer/keywords_vs_identifiers
    * For resolving the keywords vs Identifier ambiguity.
    */
-  longer_alt?: TokenType
+  longer_alt?: TokenType | TokenType[]
```

Comment:
I think its important to note in these docs that:
- Multiple longer_alts can be provided.
- The first matching one will take precedence.